ui/css/module/_footnotes.scss

```diff
@@ -31,6 +31,12 @@
     padding-bottom: 0.2em;
   }
 
+  /* It's ok to add outline: none here for accessibility purposes.
+   * For more details, see: https://youtu.be/ifW_oy9hajU */
+  .citation-wrapper[tabindex="-1"] {
+    outline: none;
+  }
+
```

Comment:
Technically I could've added this CSS to the `.citation-wrapper` rule above, but I thought adding the `[tabindex="-1"]` made it more explicit that the `outline: none` is intended _because_ the element itself is non-interactive, and we're focusing on it for reasons explained in the aforementioned [video](https://youtu.be/ifW_oy9hajU).